### ISSUE TITLE
Unable to complete baby_AFQ tutorial (viz = myafq.export("all_bundles_figure"))

### ISSUE THREAD

**Issue by chiuhoward** [yeatmanlab/pyAFQ#1021]
I built a container with miniconda with python=3.9 and 
`pip install 'git+https://github.com/yeatmanlab/pyAFQ.git#egg=pyAFQ[dev,fury,afqbrowser,plot]'`, and manual replacement of line 57 in the python3.9/site-packages/AFQ/tasks/segmentation.py file. #1017 

I’m running Apple M2 Pro with 16GB RAM on Ventura 13.5.2.

I ran the baby_AFQ tutorial both through the terminal and within a jupyter notebook on my laptop, but the process/kernel always gets terminated at the following section.

```
INFO:AFQ:Finding Streamlines for FP
INFO:AFQ:Filtering by Prob. Map
INFO:AFQ:After filtering by Prob. Map (time: 1.2395131587982178s), 229691 streamlines remain.
INFO:AFQ:Filtering by Cross Mid.
INFO:AFQ:After filtering by Cross Mid. (time: 0.0011529922485351562s), 16212 streamlines remain.
INFO:AFQ:Filtering by include
[1]    78324 killed     python baby_AFQ.py

~/AFQ_data/baby_example ......................................................................................................................................................... KILL took 20m 40s py pyAFQ1.1 at 16:43:46
```

**Comment by chiuhoward** [comment]
Of the 6 examples on the http://yeatmanlab.github.io/pyAFQ/tutorials/tutorial_examples/index.html, I only managed to complete the example on plotting tract profiles, and I did not run “How pyAFQ uses BIDS”.

It also failed for:
- “RecoBundles80 using AFQ API”
- "AFQ API”
- "Using Subject Space ROIs from Freesurfer”

which I will file separate issues for with the error logs. (hoping it’s not just a me/my local installation issue…)

**Comment by 36000** [comment]
This could be fixed by #1030
